exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetGroupScanStatistics.java

```diff
@@ -88,19 +88,21 @@ public void collect(List<RowGroupInfo> rowGroupInfos, ParquetTableMetadataBase p
       for (ColumnMetadata column : rowGroup.getColumns()) {
         SchemaPath schemaPath = SchemaPath.getCompoundPath(column.getName());
         Long previousCount = columnValueCounts.get(schemaPath);
+        Long newCount;
```

Comment:
@vvysotskyi definitely case worth fixing! 
Maybe we can optimize `if (previousCount != null)` to something like `prevoiusCount == null ? 0 : previousCount`? This would help to remove else part and duplicated logic.

Reply:
Thanks for the advice, with this change the code looks prettier!